### PR TITLE
Miscellaneous intro edits

### PR DESCRIPTION
**Pull request by dhimmel**

### CODE REVIEW

File: content/02.main-text.md [hunk 1]
```diff
@@ -3,27 +3,27 @@
...
-In total, the project that we termed the "Deep Review" attracted {{deep_review_authors}} authors from 20 different institutions (and counting) who were not determined in advance.
+In total, the Deep Review attracted {{deep_review_authors}} authors from 20 different institutions (and counting) who were not determined in advance.
 
 Writing review articles in a public forum allows review authors to engage with the original researchers to clarify their methods and results and present them accurately, as exemplified [here](https://github.com/greenelab/deep-review/issues/213).
 `TODO: need archival issue link`
-In addition, discussing manuscripts in the open provides one form of pre- and post-publication peer review `TODO: define this or provide a reference?`, incentivizing the reviews with potential manuscript authorship.
+In addition, discussing manuscripts in the open provides one form of pre- and post-publication peer review [@doi:10.7554/eLife.38532], incentivizing the reviews with potential manuscript authorship.
```
Comment (dhimmel) [inline content/02.main-text.md:26]: What does "incentivizing the reviews with potential manuscript authorship" mean? Should we say "incentivizing peer review of existing literature with Deep Review authorship" or similar? 

My compaint is that the word review is ambiguous as currently written

Comment (agitter) [reply content/02.main-text.md:26]: Yes, you have the correct sentiment.  I suggest keeping this paragraph about writing open review manuscripts in general.  "incentivizing peer review of existing literature with review manuscript authorship" is what I have in mind, but the repeated "review" is clunky.

**Review by agitter — APPROVED** [review]
Looks good.  I'll work on the ambiguous sentence that you noted when I edit the intro.

**Comment by cgreene** [comment]
I'm 👍 on this if @agitter is and it moves things forward.

**Pull request merged**